webapp/src/Components/Shared/SideBar.tsx

```diff
@@ -76,12 +74,12 @@ class SideBarComponent extends React.PureComponent<Props> {
                         <Divider component="li" />
 
                         <ListSubheader>Player</ListSubheader>
-                        <ListItem button component={this.createLink} to={LEADERBOARDS_LINK}>
+                        {/*                        <ListItem button component={this.createLink} to={LEADERBOARDS_LINK}>
```

Comment:
Maybe link a github issue to this so we don't drop it or have no tracking for it long term

Reply:
It's not going away. The front page is until I can figure out a better way to get the most recent games

https://github.com/SaltieRL/DistributedReplays/issues/458